querydsl-core/pom.xml

```diff
@@ -48,17 +48,6 @@
     </dependency>
     
      <!-- backwards compatibility -->
-    <dependency>
-      <groupId>com.infradna.tool</groupId>
```

Comment:
`bridge-method-annotation` is unmaintained and doesn't support Java 8 bytecode.

The bridge methods are also not needed anymore: its safe to remove backwards compatibility methods in a major release.